staging/src/github.com/kubeedge/api/common.go

```diff
@@ -0,0 +1,3 @@
+package apis
+
+const ISO8601UTC = "2006-01-02T15:04:05Z"
```

Comment:
Cant this constant be moved into the package `common/constants`?

Reply:
Done